web/components/templates/users/UserDocsLink.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Link from 'next/link';
+import { DocumentIcon } from '@heroicons/react/outline'; // Replace this with the actual icon import you wish to use
```

Comment:
Type error: Could not find a declaration file for module '@heroicons/react/outline'. '/vercel/path0/web/node_modules/@heroicons/react/outline/index.js' implicitly has an 'any' type.
--
00:16:58.831 | If the '@heroicons/react' package actually exposes this module, try adding a new declaration (.d.ts) file containing `declare module '@heroicons/react/outline';`
00:16:58.831 |  
00:16:58.831 | 1 \| import React from 'react';
00:16:58.832 | 2 \| import Link from 'next/link';
00:16:58.832 | > 3 \| import { DocumentIcon } from '@heroicons/react/outline'; // Replace this with the actual icon import you wish to use
00:16:58.832 | \|                              ^
00:16:58.832 | 4 \|
00:16:58.841 | 5 \| const UserDocsLink = () => {
00:16:58.841 | 6 \|     return (
00:16:58.986 | Error: Command "yarn run build" exited with 1